package.json

```diff
@@ -115,6 +116,7 @@
 		"hpagent": "1.2.0",
 		"react-markdown": "^9.0.1",
 		"semver": "^7.6.0",
+		"shell-quote": "^1.8.1",
```

Comment:
[This package](https://github.com/ljharb/shell-quote) is used to parse the CLI arguments passed as a single string to an array of strings, as need to invoke `wp-cli`.